ui/core/src/model/annotations.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Definition } from './definitions';
+import { Display } from './display';
+
+export type AnnotationEvent = {
+  timestamp: number;
+  endTimestamp?: number;
+  color?: string;
+  title?: string;
```

Comment:
nit: personally i think `description` makes more sense than `title`.  

Or can you have both `title` and `description`?

Reply:
Oh yeah not a bad idea, I think both make sense.